src/lib.rs

```diff
@@ -22,6 +22,9 @@
 use std::u32;
 use std::collections::HashMap;
 
+extern crate byteorder;
+use self::byteorder::{ByteOrder, LittleEndian};
```

Comment:
Keep the `extern crates` together and move the `use byteorder...` line below that then. The `self::` should not be necessary anymore.

Reply:
Would you prefer to keep all the `use`'s together and move the two existing`use std::` lines below the group of `extern`'s

Reply:
That's my preferable style, yes.

1. extern
2. use std
3. use external stuff